### DISCUSSION TITLE
DarkestDungeon 打不开 缺乏运行库 libssl.so.1.0.0

### DISCUSSION THREAD

**Discussion by EriaWalker** [Gamuxorg/bbs#395] [游戏或网站运行反馈] [answered]
##  问题
```
:~/Programs$ ./DarkestDungeon_25688_amd64.appimage 
architecture of your Operating System is x86_64
architecture of your Operating System is x86_64
./_linuxnosteam/darkest.bin.x86_64: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
```
## 环境
Ubuntu 22.04.1 LTS

## 已经尝试的：

1. 检查是否赋予可执行权限
chmod a+x 文件名
2. 检查fuse是否安装
以apt为例sudo apt install fuse
3. 安装steam,并至少打开一次
4. 从https://www.openssl.org/source/old/1.0.0/ 和 http://security.ubuntu.com/ubuntu/pool/main/o/openssl1.0/ 下载库，不知安装包是否有效，安装路径是否正确

## 建议：

1. 更新appimage包，提供运行库
2. 提供运行库的修复方法

## 这个游戏挺好玩，我很喜欢，恳请尽快解决，thanks ;-)

**Comment by hyh19962008** [comment, marked as answer]
先找一个有效的 libssl.so.1.0.0，有安装steam的话可以在~/.steam/steam下搜索，放到一个目录下（例如/tmp），命令行输入 
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/tmp ，然后再启动游戏

**Reply by EriaWalker** [reply]
好嘞，已解决，谢谢你啦 : )
---
似乎这游戏没有汉化.

路径：
/snap/core18/2560/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/snap/core18/2560/usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.0
